src/lang/export.pl

```diff
@@ -44,3 +44,23 @@
   tripledb_export(Directory),
   obda_export(Directory),
   forall(memorize_hook(Directory), true).
+
+     /*******************************
+     *          UNIT TESTS          *
+     *******************************/
+
+:- begin_tests('lang_export').
+
+get_path(Path):-
+  working_directory(X,X), string_concat(X, "test_lang_export", Path).
+
+test('stores knowledge in test_lang_export directory and restores the same', 
+  [ 
+    setup(get_path(Path)),
+    cleanup(shell('cd $(rospack find knowrob); rm -rf test_lang_export'))
+  ]) :-
+  assert_true(memorize(Path)),
+  assert_true(tripledb_whipe),
```

Comment:
I think this is not ok.
only the test graph in tripledb may be wiped, and is wiped automatically after each test unit.

Reply:
maybe in such a test it would be better to use a different DB in mongo. i.e. one with a different name then the one used by the other tests.
then you would have full control and could wipe the DB as part of testcases